### PR TITLE
Add QGIS

### PR DESCRIPTION
**Pull request by marcolongol**
- [X] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/qgis.json [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+            "QGIS\\QGIS Desktop"
+        ],
+        [
+            "bin\\grass78.bat",
+            "QGIS\\GRASS GIS 7.8.7",
+            "--gui",
+            "apps\\grass\\grass78\\gui\\icons\\grass.ico"
+        ],
+        [
+            "osgeo4w.bat",
```
Comment (rasa) [inline bucket/qgis.json:33]: Does this want a `bin\` prefix?

Comment (marcolongol) [reply bucket/qgis.json:33]: Nah, this one goes into the root dir

**Comment by marcolongol** [comment]
Any idea if we're even pulling this one out someday? :)

**Comment by HUMORCE** [comment]
The install script is not working with version 3.30.1.

**Pull request closed without merging**